SPHINXsys/src/shared/particle_dynamics/solid_dynamics/solid_dynamics.h

```diff
@@ -241,7 +241,13 @@ namespace SPH
 			virtual void Update(size_t index_i, Real dt = 0.0) override;
 		};
 
-				/**
+		/**
```

Comment:
Here seems not the place to declare this function.

Reply:
okey